examples/solid/default-query-function/package-lock.json

```diff
@@ -0,0 +1,1737 @@
+{
```

Comment:
I think there should only be one `package-lock.json` file at the root of the monorepo. The examples should be listed as a workspace here:

https://github.com/TanStack/query/blob/ea80700c2c8d48c6d0cb4ae0165a424ebea5d35c/package.json#L30-L56

Reply:
resolved in #4211.